ci/utils/linux-common-runnner.sh

```diff
@@ -19,7 +16,7 @@ export_or_prefix() {
 get_apisix_code() {
     # ${1} branch name
     # ${2} checkout path
-    git_branch=${1:-release/2.12}
+    git_branch=${1:-master}
```

Comment:
Shall we use LTS instead of the `master` branch ?

Reply:
@leslie-tsang No, saml-auth plugin depends on https://github.com/apache/apisix/pull/7893 on master branch.